src/imagings/search/ViewImagings.tsx

```diff
@@ -56,8 +72,34 @@ const ViewImagings = () => {
     }
   }, [getButtons, setButtons])
 
+  const onSearchBoxChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchText(event.target.value)
+  }
+
   return (
     <Container>
+      <Row>
+        <Column md={3} lg={2}>
+          <SelectWithLabelFormGroup
+            name="type"
+            label={t('imagings.filterTitle')}
+            options={filterOptions}
+            defaultSelected={filterOptions.filter(({ value }) => value === searchFilter)}
+            onChange={(values) => setSearchFilter(values[0] as ImagingFilter)}
+            isEditable
+          />
+        </Column>
+        <Column>
+          <TextInputWithLabelFormGroup
+            name="searchbox"
+            label={t('imagings.search')}
+            placeholder={t('imagings.searchPlaceholder')}
+            value={searchText}
+            isEditable
+            onChange={onSearchBoxChange}
+          />
```

Comment:
need to add the following tests related to this field: 

- make sure that it gets rendered on the page
- make sure that when its value changes that it performs a search